src/backend/joanie/core/models/accounts.py

```diff
@@ -76,9 +76,17 @@ def clean(self):
         return super().clean()
 
     def save(self, *args, **kwargs):
-        """Enforce validation each time an instance is saved."""
+        """
+        Enforce validation each time an instance is saved and trigger the
+        commercial newsletter subscription task if the user has subscribed to
+        """
         self.full_clean()
-        if self.created_on is None and self.has_subscribed_to_commercial_newsletter:
+
+        is_creating = self.created_on
```

Comment:
```suggestion
        is_creating = self.created_on is None
```
Just to get a boolean, as the name `is_creating` suggests ?
or
```suggestion
        is_creating = bool(self.created_on)
```
WDYT ?

Reply:
Yup, the variable is formatted as a boolean so let's use a boolean